_posts/2021-01-18-changelog-139.markdown

```diff
@@ -0,0 +1,115 @@
+  TODO: gif
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6582 -->
+* Add basic support for [overridden](https://rust-lang.github.io/rustup/overrides.html) toolchains.
+  Note, multiple overridden toolchains in a single project are still unsupported
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6553 -->
+* Resolve URLs in `Cargo.toml` (e.g. in dependency `git` links).
+  Now you can open URL specified in `git`, `homepage`, `repository`, and `documentation` fields (by [@Kobzol])
+  TODO: gif?
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6084 -->
+* Provide folding for TOML files. Currently, it supports tables, arrays, and custom folding regions (by [@Kobzol])
+  TODO: gif?
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6120 -->
+* Highlight [string-formatting](https://doc.rust-lang.org/rust-by-example/hello/print/fmt.html) arguments of `panic!` macro
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6439 -->
+* Incrementally update our experimental [local crates index](https://github.com/intellij-rust/intellij-rust/pull/6390).
+  This speeds up the experimental `Cargo.toml` dependency completion. In the future, this would affect other new code insight based on the local crates index as well.
+  Note, the corresponding feature is disabled by default for now.
+  You can enable it via `org.rust.crates.local.index` option in `Experimental Feature` dialog (`Help | Find Action` and type `Experimental features` to open the dialog)
+
+## Fixes
+* Cargo features and `cfg` support improvements:
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6549 -->
+    * Fairly evaluate unknown `cfg` options to `false` if there is no custom [build script](https://doc.rust-lang.org/cargo/reference/build-scripts.html) (`build.rs`) in the package.
+      Previously, in order to avoid wrongly graying out the actual code, unknown `cfg` options and unknown Cargo features were considered _enabled_ and _disabled_ simultaneously.
+      Note, you can turn on the new logic for packages containing `build.rs` as well by enabling our experimental build script evaluation:
+      just enable `org.rust.cargo.evaluate.build.scripts` in `Help | Find Action | Experimental Features` dialog
+      TODO: might be in the features block
+
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6507 -->
+    * Improve `#[cfg(test)]` processing for external dependencies by providing a new heuristic.
+      Now our code insight treats `#[cfg(test)]` as _disabled_ if there is a `cfg(not(test))` condition in the project.
+      Previously, `test` option was considered _enabled_ and _disabled_ simultaneously, i.e. code under `#[cfg(test)]` and `#[cfg(not(test))]` was both enabled.
+      Such test attributes are quite complex to handle in the IDE because we do not want to gray out all the tests, but do want to gray out only the mock objects.
```

Comment:
"Gray out" is not clear enough, at least for me. I suppose we should use another term for such code. "Conditionally disabled" for example

Reply:
Sounds right. I replaced it with "to conditionally disable (exclude from analysis and gray out)", as IMO it's clear enough for both experienced users and newbies.